diff --git a/dist/providers/Auth/index.js b/dist/providers/Auth/index.js
index b380e428b2331d508c97033b1b7c1577782baf41..a0dd3dfbb663bef3f520b62ae3c6ff5eaa65695c 100644
--- a/dist/providers/Auth/index.js
+++ b/dist/providers/Auth/index.js
@@ -54,7 +54,7 @@ export function AuthProvider({
   const [tokenExpirationMs, setTokenExpirationMs] = useState();
   const [permissions, setPermissions] = useState(initialPermissions);
   const [forceLogoutBufferMs, setForceLogoutBufferMs] = useState(120_000);
-  const [fetchedUserOnMount, setFetchedUserOnMount] = useState(false);
+  const [fetchedUserOnMount, setFetchedUserOnMount] = useState(true);
   const refreshTokenTimeoutRef = React.useRef(null);
   const reminderTimeoutRef = React.useRef(null);
   const forceLogOutTimeoutRef = React.useRef(null);
